File: Java/Random.java
import java.util.Date;
import java.util.concurrent.TimeUnit;

class RandomGenerator{
    public static void main(String[] args) {
        System.out.println(randomRange(1,10));
    }

    static long generateSeed(){
        long x = TimeUnit.NANOSECONDS.toMicros(System.nanoTime());;
        x ^= x >> 12;
        x ^= x << 25;
        x ^= x >> 27;
        return x * 268582165;
    }
    
    public static int randomRange(int min, int max){
        int n = max - min + 1;
        int remainder = Integer.MAX_VALUE % n;
        int x;
        do{
            x = (int)generateSeed();
        } while (x >= Integer.MAX_VALUE - remainder);
    
        if((min + x % n) < min){
            return Math.abs((min + x % n));
        }
        else{
            return (min + x % n);
        }
    }
}

File: Python/Random.py
import math
from datetime import datetime

def generateSeed():
    dt = datetime.now()
    x = dt.microsecond
    x ^= x >> 12
    x ^= x << 25
    x ^= x >> 27
    return x * 2685821657736338717

def random_range(min, max):
    n = max - min + 1
    remainder = math.inf % n
    x = generateSeed()
    
    while x >= math.inf - remainder:
        x = generateSeed()

    if (min + x % n) < min:
        return abs(min + x % n)
    else:
        return min + x % n
        
print(random_range(1, 10))
